backends/platform/libretro/scripts/configure_engines.sh

```diff
@@ -108,7 +108,7 @@ for comp in $(get_var _components); do
 done
 
 # Create needed engines build files
-awk -f "engines.awk" < /dev/null > /dev/null 2>&1
+awk -f "${SCUMMVM_PATH}/engines.awk" < /dev/null > /dev/null 2>&1
```

Comment:
We change to `${SCUMMVM_PATH}` at the beginning of the script. Why is this needed?